### ISSUE TITLE
Envoy docs may have restructured

### ISSUE THREAD

**Issue by GregHanson** [istio/istio.io#13494]
`Access Log Service` and `Metrics Service` links are returning 404 on https://istio.io/latest/docs/reference/config/istio.mesh.v1alpha1/:
 - https://www.envoyproxy.io/docs/envoy/latest/api-v2/config/accesslog/v2/als.proto
 - https://www.envoyproxy.io/docs/envoy/latest/api-v2/config/metrics/v2/metrics_service.proto

Assuming next person to submit a docs PR will get some link test failures - not sure if any other envoy doc links have been impacted

**Comment by ericvn** [comment]
I don't think we test a lot of external links (disabled long long ago), and this could be one of them. Would need to investigate more.

**Comment by GregHanson** [comment]
regarding testing the external links - what was the discussion behind removing them?  I could see not running the tests on every pull request - but would it be a worthwhile once a month job or something?  

Or is it just a case of we can't expect non-istio managed links from X years ago to still work?

**Comment by ericvn** [comment]
Also noted in Slack: `https://istio.io/latest/docs/reference/config/istio.mesh.v1alpha1/#ForwardClientCertDetails` has a broken Envoy link.

**Comment by rsalmond** [comment]
I looked into updating those two links while looking into istio/api#2937. The metrics_service.proto docs haven't changed much between [v2](http://web.archive.org/web/20210417141846/https://www.envoyproxy.io/docs/envoy/latest/api-v2/config/metrics/v2/metrics_service.proto) and [v3](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/metrics/v3/metrics_service.proto) but the logging service docs ([v2](https://web.archive.org/web/20210204170645/https://www.envoyproxy.io/docs/envoy/latest/api-v2/config/accesslog/v2/als.proto) [v3](https://www.envoyproxy.io/docs/envoy/latest/api-v3/service/accesslog/v3/als.proto) are quite different.

Are these references still pertinent?
